CalibCalorimetry/EcalLaserAnalyzer/plugins/EcalABAnalyzer.cc

```diff
@@ -73,8 +67,10 @@ EcalABAnalyzer::EcalABAnalyzer(const edm::ParameterSet& iConfig)
       _noise(iConfig.getUntrackedParameter<double>("noise", 2.0)),
       _chi2cut(iConfig.getUntrackedParameter<double>("chi2cut", 100.0)),
       _ecalPart(iConfig.getUntrackedParameter<std::string>("ecalPart", "EB")),
+      _fedid(iConfig.getUntrackedParameter<int>("fedId", -999)),
       _qualpercent(iConfig.getUntrackedParameter<double>("qualPercent", 0.2)),
       _debug(iConfig.getUntrackedParameter<int>("debug", 0)),
+      resdir_(iConfig.getUntrackedParameter<std::string>("resDir")),
```

Comment:
There is an internal naming inconsistency here as well, right? what was here earlier all started with an underscore, but the new code doesn't and in case of the `resdir_` the underscore is in the end. Could this be made uniform in the code? Thanks!

Reply:
I agree that this looks rather ugly. The inconsistency in member variable naming scheme has been there before. The initializer list has just made it more apparent. Fixing all these variables will take significant time that I think is better spent migrating more packages to esConsumes.